src/unsteady_ns_solver.cpp

```diff
@@ -88,6 +88,11 @@ bool UnsteadyNSSolver::Solve(SampleGenerator *sample_generator)
    std::string restart_file, file_fmt;
    file_fmt = "%s/%s_%08d.h5";
 
+   StopWatch timers;
+   for (int k = 0; k < 10; k++)
```

Comment:
Instead of hard-coding `10`, it may be better to make a `constexpr int` in the class. An `mfem::Vector` could be used instead of `double[10]`.